pkg/cloud/aws/aws.go

```diff
@@ -56,25 +56,26 @@ type machineSetConfig struct {
 }
 
 type awsProvider struct {
-	workClinet    workclient.Interface
-	awsClinet     client.Interface
-	eventRecorder events.Recorder
-	region        string
-	infraId       string
-	ikePort       int64
-	nattPort      int64
-	clusterName   string
-	instanceType  string
-	gateways      int
-	cloudPrepare  cpapi.Cloud
+	workClinet        workclient.Interface
```

Comment:
```suggestion
	workClient        workclient.Interface
```
small typo

Reply:
@mangelajo This is used in about 20 different places in code. IMO this change should be done in a separate PR to avoid cluttering this one. thoughts?